src/edu/washington/escience/myria/operator/network/partition/IdentityHashPartitionFunction.java

```diff
@@ -0,0 +1,57 @@
+package edu.washington.escience.myria.operator.network.partition;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Preconditions;
+
+import edu.washington.escience.myria.Type;
+import edu.washington.escience.myria.storage.TupleBatch;
+
+/**
+ * Implementation of a PartitionFunction that use the trivial identity hash.
```

Comment:
Could you make it explicit that the relation must have an `INT` column that stores `workerID`s?